config.properties

```diff
@@ -1,36 +1,44 @@
-# Copyright (c) 2021-2022, Adel Noureddine, Universit� de Pays et des Pays de l'Adour.
-# All rights reserved. This program and the accompanying materials
-# are made available under the terms of the
-# GNU General Public License v3.0 only (GPL-3.0-only)
-# which accompanies this distribution, and is available at
-# https://www.gnu.org/licenses/gpl-3.0.en.html
-#
-# Author : Adel Noureddine
-
-# Get also power and energy for methods starting with this package.class.method name
-# Example: filter-method-names=Package to filter energy for all methods whose full name start with Package
-# Note that JoularJX will always generate energy for all non-filtered method in a separate file
-# So you can safely keep this empty if you don't want to filter methods
-# You can add multiple identifiers names separated by a comma, ordered by priority
-# JoularJX will monitor and filter methods for all these packages
-# Example: filter-method-names=Package,Package2,Package3
-filter-method-names=Package
-
-# Write runtime methods power consumption in a CSV file
-# Setting this option to false won't generate any runtime files
-# Allowed values: true, false
-save-runtime-data=true
-
-# Overwrite runtime power data file
-# Setting it to false will write files for every monitoring cycle (1 sec per cycle)
-# Allowed values: true, false
-overwrite-runtime-data=true
-
-# Logging level
-# Set to OFF to disable it
-# Allowed values: OFF, INFO, WARNING, SEVERE
-logger-level=INFO
-
-# Path for our power monitor program on Windows
-# Please escape slashes twice
+# Copyright (c) 2021-2022, Adel Noureddine, Universit� de Pays et des Pays de l'Adour.
+# All rights reserved. This program and the accompanying materials
+# are made available under the terms of the
+# GNU General Public License v3.0 only (GPL-3.0-only)
+# which accompanies this distribution, and is available at
+# https://www.gnu.org/licenses/gpl-3.0.en.html
+#
+# Author : Adel Noureddine
+
+# Get also power and energy for methods starting with this package.class.method name
+# Example: filter-method-names=Package to filter energy for all methods whose full name start with Package
+# Note that JoularJX will always generate energy for all non-filtered method in a separate file
+# So you can safely keep this empty if you don't want to filter methods
+# You can add multiple identifiers names separated by a comma, ordered by priority
+# JoularJX will monitor and filter methods for all these packages
+# Example: filter-method-names=Package,Package2,Package3
+filter-method-names=Package

```

Comment:
Carriage return (^M) is present in a few occurrences in the file. Please correct it.

Reply:
Done. I also fixed the "é" at the beginning of the file